tests/test.yargs-options.js

```diff
@@ -0,0 +1,11 @@
+import { getDefaultConfigValue } from 'yargs-options';
+
+describe('getDefaultConfigValue()', () => {
+  it('should return the default value', () => {
+    expect(getDefaultConfigValue('self-hosted')).toBeFalsy();
```

Comment:
Let's change this to use `.toEqual(false)` if we are expecting this or `undefined` etc depending on what value we expect. `falsy` can be many things and explicit is better than implicit.

Reply:
Thanks.  That's good to know. I didn't know that what 'toBeFalsy()' exactly does.